src/Header/Header.tsx

```diff
@@ -5,12 +5,14 @@ import './Header.scss';
 interface HeaderProps {
 	loggedIn: boolean;
 	setLoggedIn: any;
+	setUsername: Function
 }
 
 const Header: React.SFC<HeaderProps> = props => {
 
 	const logoutUser = () => {
 		props.setLoggedIn(false);
+		props.setUsername('')
```

Comment:
resets username on a logout so the login input holds the correct blank value

Reply:
1+ for reseting forms!